ServerCore/Pages/Teams/Play.cshtml

```diff
@@ -36,6 +36,16 @@
         <input id="StateFilter" name="StateFilter" value="Unsolved" type="radio" @(unsolvedFilter ? "checked" : "")> Unsolved
     </a>
 </div>
+<br/>
+    <a asp-page="/Events/FastestSolves"
+       asp-route-sort="CountDescending"
+       asp-route-stateFilter="Unsolved">
+
+        Feeling stuck?
+    </a>
+<br/>
+    The link above will take you to a page where you can view puzzles that have been already solved by many other teams.
```

Comment:
"where you can view puzzles, sorted by how many teams have solved them" is gentler.